test/NuGet.Core.Tests/NuGet.Commands.Test/BuildAssetsUtilsTests.cs

```diff
@@ -7,6 +7,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 using System.Xml.Linq;
+using NuGet.Commands.Test.RestoreCommandTests;
```

Comment:
Is this still needed?

Reply:
Yes. This file uses a type in the `RestoreCommandTests` subdirectory, for which I moved into a namespace matching its directory layout: https://github.com/NuGet/NuGet.Client/pull/5297#discussion_r1258824516